### DISCUSSION TITLE
Leaflet Integration

### DISCUSSION THREAD

**Discussion by ghost1w1** [javalent/obsidian-leaflet#415] [Q&A] [answered]
Hello, 
when I try displaying a battle map for the initative tracker a Leaflet Map is created but Markers are not created and I unfortunately am helpless on what to do to get it to work. 
Help would be appreciated.

Thank you very much.

**Comment by sigrunixia** [comment, marked as answer]
The battle map for Obsidian Initiative Tracker was depreciated, and final functionality is being removed in https://github.com/javalent/initiative-tracker/pull/204. We may revisit this in the future when we can guarantee more stable function (might need some obsidian upgrades D:).

**Comment by wendigo87** [comment]
[Leaflet integration](http://wendigodistribution.com/) offers seamless mapping capabilities for websites and applications. Leveraging Leaflet, an open-source JavaScript library, developers can easily embed interactive maps.
